test/ros_parameter_handler_test.cpp

```diff
@@ -547,36 +531,6 @@ TEST_F(ROSParameterHandlerTest, test_invalid_params_x_axis_rotation)
   ASSERT_NO_THROW(RosParameterHandler param_handler(node_handle));
 }
 
-TEST_F(ROSParameterHandlerTest, test_invalid_params_publish_topic)
```

Comment:
Is there a test that will make sure the node publishes to the `/scan` topic?

Reply:
[Here](https://github.com/PilzDE/psen_scan/blob/melodic-devel/test/ros_scanner_node_test.cpp#L108)
kind of ...

Reply:
I would say, this is a unit test.
I think we also need an integration test that actually runs checks the topic. 
This would serve as a regression test and make sure that the `scan` topic is in future still available to not break users applications.

Reply:
Does 9c4124a859843bdef725d0d653c8ff9312a892de solve that?

Reply:
yes :+1: